x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/LimitOperatorTests.java

```diff
@@ -19,6 +19,11 @@
 import static org.hamcrest.Matchers.equalTo;
 
 public class LimitOperatorTests extends OperatorTestCase {
+    @Override
+    protected DriverContext driverContext() {
+        return breakingDriverContext();
+    }
+
```

Comment:
Enabling this driver context ensures that this change is verified by testSimpleFinishClose